packages/services/src/models/Etherscan.js

```diff
@@ -34,8 +34,8 @@ export default class Etherscan {
   }
 
   _getBaseUrl(network) {
-    if (network === 'rinkeby' || network === 'ropsten') return BASE_URL.replace('<network>', network)
-    else if (network === 'staging') return BASE_URL.replace('<network>', 'rinkeby')
+    if (network === 'rinkeby' || network === 'ropsten') return BASE_URL.replace('<network>', `api-${network}`)
+    else if (network === 'staging') return BASE_URL.replace('<network>', 'api-rinkeby')
```

Comment:
So I assume this wasn't working at all for the test networks—should we have seen error logs?

Reply:
Both work actually, but the one specified in the documentation is the one with the `api-` prefix :)